src/ralph/accounts/management/commands/ldap_sync.py

```diff
@@ -66,7 +66,7 @@ def get_nested_groups():
     users_groups = defaultdict(set)
     nested_groups = getattr(settings, 'AUTH_LDAP_NESTED_GROUPS', None)
     if not nested_groups:
-        return
+        return None, None
```

Comment:
maybe it'll be better to return here `return group_users, users_groups`? Then it'll have types of the valid result